### PR TITLE
#17249 Fix ECR Vulnerabilities: thecodingmachine/gotenberg

### PR DESCRIPTION
**Pull request by michael-jong-ds**
Merge after https://github.com/datasaur-ai/helm-charts/pull/5 
didn't realize until now that the branch name is mixed up :facepalm: my bad

- Updated Gotenberg to version 7.8.3
- Added version requirement to 6.20.0

### CODE REVIEW

**Review by tonoman3g — APPROVED** [review]

**Comment by tonoman3g** [comment]
@daniel-jeremia mau bantu review? so far oke sih, no big deal

File: charts/datasaur/Chart.yaml [hunk 1]
```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.5.0
+version: 1.5.1
```
Comment (daniel-jeremia) [inline charts/datasaur/Chart.yaml]: Better to be 1.6.0 since there's breaking changes

Comment (tonoman3g) [reply charts/datasaur/Chart.yaml]: oh yes, i forgot about this. wkt itu gw bilangnya di slack jg 1.6.0

tolong ya @michael-jong-ds

Comment (michael-jong-ds) [reply charts/datasaur/Chart.yaml]: updated to 1.6.0 :pray:

**Review by tonoman3g — APPROVED** [review]

**Pull request merged**